daphne/src/aborts.rs

```diff
@@ -134,9 +137,8 @@ impl DapAbort {
                 Some("The request indicates an aggregation job that does not exist.".into()),
                 Some(agg_job_id_base64url),
             ),
-            Self::ReportTooLate | Self::UnrecognizedMessage | Self::UnrecognizedTask => {
-                (None, None, None)
-            }
+            Self::UnrecognizedMessage { detail, task_id } => (task_id, Some(detail), None),
```

Comment:
We could move this to the arm above to make the code less redundant:
```
            | Self::QueryMismatch { detail, task_id }
            | Self::UnauthorizedRequest { detail, task_id }
            |  Self::UnrecognizedMessage { detail, task_id }  => (Some(task_id), Some(detail), None),
```

Reply:
I'm not sure I can do that here since the `task_id` field in `UnrecognizedMessage` is actually of type `Option<TaskID>`, while it is of type `TaskID` for the other structs.

Reply:
Oh you're right, good catch :)